Makefile

```diff
@@ -1,3 +1,3 @@
-all: src/*.cpp inc/*.hpp
```

Comment:
it shouldn't be removed. These are dependencies that tells make then it should rebuild the application. So when any of inc/*.hpp file is changed, then the binary should be rebuild. Right now it will not keep proper dependencies.